### PR TITLE
feat: ML-204 - Add routes to applet tabs, add breadcrumbs to applet tabs

### PR DESCRIPTION
**Pull request by IhorNesterenko99**

### CODE REVIEW

File: src/components/Schedule/Schedule.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { useBaseBreadcrumbs } from 'hooks';
+
+export const Schedule = (): JSX.Element => {
+  const { id } = useParams();
+  const { t } = useTranslation('app');
+  const dispatch = useAppDispatch();
+  const baseBreadcrumbs = useBaseBreadcrumbs();
+
+  useEffect(() => {
+    if (id && baseBreadcrumbs && baseBreadcrumbs.length > 0) {
```
Comment (kdanilenka32) [inline src/components/Schedule/Schedule.tsx:17]: you can use     if (id && baseBreadcrumbs?.length > 0) {

File: src/pages/Applet/More/More.tsx [hunk 2]
```diff
@@ -1,7 +1,53 @@
...
+            label: t('more'),
+          },
+        ]),
+      );
+    }
+  }, [baseBreadcrumbs, id, dispatch, t, location]);
+
+  useEffect(() => {
+    const { pathname } = location;
+    pathname.includes(appletPages.addUser) ? setActiveAddUser(true) : setActiveAddUser(false);
```
Comment (kdanilenka32) [inline src/pages/Applet/More/More.tsx:43]: ypu can simplify setActiveAddUser(pathname.includes(appletPages.addUser))

File: src/components/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -17,17 +17,22 @@ export const Tabs = ({ tabs }: TabsProps): JSX.Element => {
...
+            key={labelKey}
+            icon={tabIndex === index ? activeIcon : icon}
+            label={t(labelKey)}
+            onClick={onClick}
+          />
         ))}
       </StyledTabs>
       {tabs.map(({ content }, i) => (
-        <TabPanel key={i} value={tabIndex} index={i}>
+        <TabPanel key={i} value={activeTab ? activeTab : tabIndex} index={i}>
```
Comment (kdanilenka32) [inline src/components/Tabs/Tabs.tsx:35]: y can do activeTab || tabIndex

File: src/components/Tabs/Tabs.tsx [hunk 4]
```diff
@@ -17,17 +17,22 @@ export const Tabs = ({ tabs }: TabsProps): JSX.Element => {
   return (
     <>
       <StyledTabs
-        value={tabIndex}
+        value={activeTab ? activeTab : tabIndex}
```
Comment (kdanilenka32) [inline src/components/Tabs/Tabs.tsx:20]: y can do activeTab || tabIndex

File: src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 5]
```diff
@@ -37,22 +41,26 @@ export const Breadcrumbs = (): JSX.Element => {
...
+                <StyledLabelMedium color={variables.palette.on_surface}>{label}</StyledLabelMedium>
+              )}
             </StyledBox>
           ) : (
-            <StyledLink key={index} to={crumb.navPath ? crumb.navPath : ''}>
-              {getBreadcrumbIcon(crumb.icon, crumb.label)}
-              <StyledTitleSmall fontWeight="regular" color={variables.palette.on_surface_variant}>
-                {crumb.label}
-              </StyledTitleSmall>
+            <StyledLink key={index} to={navPath ? navPath : ''}>
```
Comment (kdanilenka32) [inline src/components/Breadcrumbs/Breadcrumbs.tsx:59]: to={navPath || ""}

File: src/components/Breadcrumbs/Breadcrumbs.styles.ts [hunk 6]
```diff
@@ -33,19 +33,16 @@ export const StyledPlaceholder = styled(StyledFlexAllCenter)`
...
   transition: background-color 0.3s;
   border-radius: ${variables.borderRadius.lg};
 
   &:hover {
-    background-color: ${variables.palette.primary50_alfa5};
-
-    p {
-      color: ${variables.palette.primary};
-    }
+    background-color: rgba(66, 71, 78, 0.08);
```
Comment (kdanilenka32) [inline src/components/Breadcrumbs/Breadcrumbs.styles.ts:41]: could we move this color to variables

File: src/components/Breadcrumbs/Breadcrumbs.tsx (hunk 5 above)
Comment (IhorNesterenko99) [reply src/components/Breadcrumbs/Breadcrumbs.tsx:59]: done

File: src/components/Schedule/Schedule.tsx (hunk 1 above)
Comment (IhorNesterenko99) [reply src/components/Schedule/Schedule.tsx:17]: done

File: src/components/Tabs/Tabs.tsx (hunk 4 above)
Comment (IhorNesterenko99) [reply src/components/Tabs/Tabs.tsx:20]: done

File: src/components/Tabs/Tabs.tsx (hunk 3 above)
Comment (IhorNesterenko99) [reply src/components/Tabs/Tabs.tsx:35]: done

File: src/pages/Applet/More/More.tsx (hunk 2 above)
Comment (IhorNesterenko99) [reply src/pages/Applet/More/More.tsx:43]: done

**Review by nmialik — APPROVED** [review]

**Review by kdanilenka32 — APPROVED** [review]

**Pull request merged**